python/heterocl/rptdisp.py

```diff
@@ -0,0 +1,338 @@
+import pandas as pd
```

Comment:
Maybe just merge this file with `report.py` unless there're some reasons.

Reply:
Indeed merging them would make it clearer but I thought since the class definition extended quite a few lines, I thought having separate files would be better.

Reply:
> the class definition extended quite a few lines
What does that mean?

Reply:
> What does that mean?
I thought keeping them separate would make the `report.py` cleaner in a sense that the core function definitions of `report.py` don't get pushed towards the bottom of the script by the class definition.

Reply:
Nah. It's fine. Just merge them. Displaying the output is also part of the report.